app/controllers/admin/projects_controller.rb

```diff
@@ -31,12 +27,11 @@ def update
   end
 
   def edit
-    @request_types = RequestType.order(name: :asc)
-    if params[:id] != '0'
```

Comment:
@request_types was unused.
Rest is just making rubocop happy by flipping the condition around.